app/src/main/java/com/twilio/video/app/ui/room/RoomViewState.kt

```diff
@@ -11,13 +13,18 @@ data class RoomViewState(
     val participantThumbnails: List<ParticipantViewState>? = null,
     val selectedDevice: AudioDevice? = null,
     val availableAudioDevices: List<AudioDevice>? = null,
-    val isLobbyLayoutVisible: Boolean = true,
-    val isConnectingLayoutVisible: Boolean = false,
-    val isConnectedLayoutVisible: Boolean = false,
+    val layoutState: LayoutState = Lobby,
```

Comment:
A state within a state reads a little confusing. Can we rename this to `configuration` and `RoomViewConfiguration`?